code/images/SCIENCE.html

```diff
@@ -0,0 +1,165 @@
+<!DOCTYPE html>
+<!-- saved from url=(0037)http://127.0.0.1:5500/code/index.html -->
```

Comment:
I'm not sure how this appears because it is not in my GitHub code index.html . also the navigation bar bootstrap links and font awesome links is present in GitHub and inspect tool